src/lints/inherent_method_must_use_added.ron

```diff
@@ -0,0 +1,80 @@
+    reference_link: Some("https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-must_use-attribute"),
+    query: r#"
+    {
+        CrateDiff {
+            current {
+                item {
+                    ... on ImplOwner {
+                        visibility_limit @filter(op: "=", value: ["$public"]) @output
+                        name @tag @output
+                        owner_type: __typename @tag
+
+                        importable_path {
+                            path @tag @output
+                        }
+
+                        inherent_impl {
+                            method {
+                                method_visibility: visibility_limit @filter(op: "=", value: ["$public"]) @output
+                                method_name: name @tag @output
+
+                                attribute {
+                                    new_attr: raw_attribute @output
+                                    content {
+                                        base @filter(op: "=", value: ["$must_use"])
+                                    }
+                                }
+
+                                span_: span @optional {
+                                    filename @output
+                                    begin_line @output
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            baseline {
+                item {
+                    ... on ImplOwner {
+                        visibility_limit @filter(op: "=", value: ["$public"])
+                        name @filter(op: "=", value: ["%name"])
+                        __typename @filter(op: "=", value: ["%owner_type"])
+
+                        importable_path {
+                            path @filter(op: "=", value: ["%path"])
+                        }
+
+                        inherent_impl {
+                            method {
+                                visibility_limit @filter(op: "=", value: ["$public"])
+                                name @filter(op: "=", value: ["%method_name"])
+
+                                attribute @fold @transform(op: "count") @filter(op: "=", value: ["$zero"]) {
+                                    content {
+                                        base @filter(op: "=", value: ["$must_use"])
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }"#,
+    arguments: {
+        "public": "public",
+        "must_use": "must_use",
+        "zero": 0,
+    },
+    error_message: "An inherent method is now #[must_use]. Downstream crates that did not use its return value will get a compiler lint.",
+    per_result_error_template: Some("inherent method {{join \"::\" path}}::{{method_name}} in {{span_filename}}:{{span_begin_line}}"),
```

Comment:
```suggestion
    per_result_error_template: Some("method {{join \"::\" path}}::{{method_name}} in {{span_filename}}:{{span_begin_line}}"),
```
I think we can afford to omit `inherent` here, in the interest of keeping this message shorter:
- It makes the span info appear earlier, and following the span info will get the reader to the right method regardless of whether they've remembered that the method is inherent or not.
- The lint name and error message already contain "inherent" in the name, so the odds are good the user has noticed at least one of them anyway.